multivault-dotted-vars.yml

```diff
@@ -0,0 +1,7 @@
+dotted: !vault |
+          $ANSIBLE_VAULT;1.1;AES256;dotted.id
+          32363764366261386339653030393130623731306264353133333163393331613337383936373331
+          3938366439366135353038626665663434343738363831370a303463623433376165633831616666
+          31333332346336346533656462353634646265626363333637623931313837363437346461666635
+          3964316165373561390a323632633764646166363130623934346364363533366635313332623236
+          6432
```

Comment:
maaaybe it would be nice to put this in `utils/` folder so it won't be in the top-level namespace?

Reply:
Agreed and done.